bin/pycbc_live

```diff
@@ -553,9 +556,9 @@ class LiveEventManager(object):
         comment = comment.format(
             ppdets(coinc_ifos),
             args.ranking_statistic,
-            set(ifos) - self.skymap_only_ifos,
-            ppdets(followup_ifos),
-            ppdets(self.skymap_only_ifos)
+            ppdets(set(ifos) - self.skymap_only_ifos),
```

Comment:
ok, so 'ifos' here is all ifos involved in some way in the event?  is this going to be different from live_ifos?